test/e2e/e2e_test.go

```diff
@@ -293,32 +273,29 @@ func verifyResources(t *testing.T, awaitility *wait.Awaitility, murName string,
 		UserAccountStatus: userAccount.Status,
 	}
 	_, err = hostAwait.WaitForMasterUserRecord(mur.Name, append(masteruserrecordCriteria, wait.UntilMasterUserRecordHasUserAccountStatuses(uaStatus))...)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 
 	_, err = memberAwait.WaitForUser(userAccount.Name)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 
 	_, err = memberAwait.WaitForIdentity(toIdentityName(userAccount.Spec.UserID))
-	assert.NoError(t, err)
-
-	_, err = memberAwait.WaitForNSTmplSet(userAccount.Name)
-	assert.NoError(t, err)
-}
-
-func createMasterUserRecord(t *testing.T, awaitility *wait.Awaitility, ctx *framework.TestCtx, name string) *toolchainv1alpha1.MasterUserRecord {
-	memberCluster, ok, err := awaitility.Host().GetKubeFedCluster(cluster.Member, wait.ReadyKubeFedCluster)
 	require.NoError(t, err)
-	require.True(t, ok, "KubeFedCluster should exist")
-	mur := murtest.NewMasterUserRecord(name,
-		murtest.MetaNamespace(awaitility.HostNs), murtest.TargetCluster(memberCluster.Name))
 
-	err = awaitility.Client.Create(context.TODO(), mur, testsupport.CleanupOptions(ctx))
+	_, err = memberAwait.WaitForNSTmplSet(userAccount.Name)
 	require.NoError(t, err)
 
-	verifyResources(t, awaitility, mur.Name,
-		wait.UntilMasterUserRecordHasConditions(provisioned()),
-		wait.UntilUserAccountHasConditions(provisioned()))
-	return mur
+	// Verify all namespaces and RoleBindings in these namespaces
+	assert.Len(t, expectedRevisions, 3)
+	for key, revsion := range expectedRevisions {
```

Comment:
See https://github.com/codeready-toolchain/toolchain-e2e/commit/77c23fb4b10732473b141719e50b61cc4459d9fb#r36021931

Reply:
`revsion` -> `revision` type fixed in https://github.com/codeready-toolchain/toolchain-e2e/pull/49/commits/4bc3c4ceb26cc2372a2a640229989c0627922105